src/Applications/LDAS_App/util/compress_nc.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/local/bin/bash
+#
+# compress bit-shaved nc output
+
+usage(){
+	echo -e "\nUsage: $0 <directory>
+	<directory> = name of output directory with bit-shaved nc files
+"
+	exit 1;
+}
+
+if [[ $# -lt 1 ]]; then
+	usage
+	exits;
+fi
+
+# deflation_level 1-9; higher level (e.g. 9) takes longer; recommended: 3 (based on L4_SM) 
+deflate_level=3
+
+for file in $(find $1 -name "*.nc4" -type f); do
+	#echo "Processing $file..."
+	ncks -L $deflate_level -O $file $file &
```

Comment:
@weiyuan-jiang:  I'm concerned the ampersand at the end of the `ncks` command will create problems if there are many files in the directory.  If there are, say, 100 nc4 files, then 100 ncks commands would be kicked off nearly instantaneously, which may bog down the login node.  For multi-threading, I think we would need a more sophisticated approach that limits the number of simultaneous ncks jobs to, say, 50% of the number of available processors.  I imagine this isn't trivial, and it may be simpler just to run the ncks jobs sequentially.  Thoughts?

Reply:
usually if a users has many jobs, he should not do it on login nodes.  If time is not an issue, sequential jobs is totally fine.